plugins/rocks.xml.webhelp/css/topic.css

```diff
@@ -35,6 +35,13 @@
 :root {
     --bs-font-sans-serif: 'Montserrat', sans-serif;
 }
+
+h1.title {
```

Comment:
What about 2-nd+ level topic title? Please update input content for testing by adding chunk attr.

Reply:
@ibarinova  We need to discuss this together